inc/connector/namespace.php

```diff
@@ -138,14 +161,28 @@ function pre_schedule_event( $pre, $event ) {
  *     @type array        $args      Array containing each separate argument to pass to the hook's callback function.
  *     @type int          $interval  The interval time in seconds for the schedule. Only present for recurring events.
  * }
+ * @param bool $wp_error Optional. Whether to return a WP_Error on failure. Default false.
  * @return bool True if event successfully rescheduled. False for failure.
  */
-function pre_reschedule_event( $pre, $event ) {
+function pre_reschedule_event( $pre, $event, $wp_error = false ) {
 	// Allow other filters to do their thing.
 	if ( $pre !== null ) {
 		return $pre;
 	}
 
+	$schedules = wp_get_schedules();
+
+	if ( ! isset( $schedules[ $event->schedule ] ) ) {
```

Comment:
I think `$event->schedule` can be boolean false here for a non-recurring event, which means this would prevent `wp_reschedule_event()` from working on a non-recurring event.

Reply:
Ah sorry this is not true, the schedule is always a string at this point. The docs for the `pre_reschedule_event` in WP core aren't accurate.

Reply:
`wp_reschedule_event()` is only recurring events only.